libraries/chain/include/golos/chain/steem_object_types.hpp

```diff
@@ -59,8 +59,8 @@ namespace golos {
             feed_history_object_type,
             convert_request_object_type,
             liquidity_reward_balance_object_type,
-            operation_object_type,
-            old_account_history_object_type,
+            // old_operation_object_type,
+            // old_account_history_object_type,
             category_object_type,
```

Comment:
Remove `category_object_type` - it doesn't exist any more.

Reply:
Done